crates/forge/src/result.rs

```diff
@@ -395,29 +396,39 @@ impl fmt::Display for TestResult {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         match self.status {
             TestStatus::Success => "[PASS]".green().fmt(f),
-            TestStatus::Skipped => "[SKIP]".yellow().fmt(f),
+            TestStatus::Skipped => {
+                let mut s = String::from("[SKIP");
+                if let Some(reason) = &self.reason {
+                    write!(s, ": {reason}").unwrap();
+                }
+                s.push(']');
+                s.yellow().fmt(f)
+            }
             TestStatus::Failure => {
```

Comment:
only functional change here is removing the unconditional `. Reason: ` and "assertion failed" placeholder